setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+
+from rna_cd import VERSION
```

Comment:
I am not a fan of importing the code you are packaging and let that influence how the code is packaged. But I understand that this is common practice in the python world. 

([OPINION]Does not make it any less WRONG![/OPINION]  \*\*mutters something about abominations\*\*